course_access_groups/urls.py

```diff
@@ -4,21 +4,34 @@
 """
 from __future__ import absolute_import, unicode_literals
 
-from django.conf.urls import include, url
 from rest_framework import routers
-
 from course_access_groups import views
 
 
-router = routers.DefaultRouter()
+router = routers.SimpleRouter()
 
 router.register(
     r'groups',
-    views.GroupsViewSet,
+    views.GroupViewSet,
     base_name='groups',
 )
 
+router.register(
+    r'memberships',
+    views.MemberViewSet,
+    base_name='memberships',
+)
+
+router.register(
+    r'rules',
+    views.RuleViewSet,
+    base_name='rules',
+)
+
+router.register(
+    r'course-groups',
+    views.CourseGroupViewSet,
+    base_name='course-groups',
+)
 
-urlpatterns = [
-    url(r'^api/course_access_groups/v1/', include(router.urls, namespace='course-access-groups-api')),
```

Comment:
I'll get the `api/course_access_groups/v1/` prefix back inside the `apps.py` when integrating with the Open edX platform.